src/isPalindrome.js

```diff
@@ -19,7 +19,20 @@
  * @return {boolean}
  */
 function isPalindrome(str) {
-  // write code here
+  if (!str) {
+    return true;
+  }
+
+  const readyStr = str.toLocaleLowerCase().match(/[a-zA-Z]/g).join('');
```

Comment:
- what is `ready`?
- do not use `Locale`
- use `i` (ignore case)
```suggestion
  const replacedString = str.toLowerCase().match(/[a-z]/gi).join('');
```